src/general/StageHUDBase.cs

```diff
@@ -757,7 +757,7 @@ public void ShowFossilisationDialog(FossilisationButton button)
         if (button.AttachedOrganism is Microbe microbe)
         {
             fossilisationDialog.SelectedSpecies = microbe.Species;
-            fossilisationDialog.Show();
+            fossilisationDialog.PopupCenteredShrink();
```

Comment:
Here I made the fossilization dialog be shown according to the [style guide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#:~:text=Popups%20should%20be%20shown%20with%20PopupCenteredShrink()) as I think it's the correct way what's intended to be done here.

Past observations show that Godot popups don't usually resize dynamically, this is bad when switching between languages as text lengths differ. This is the main reason why `PopupCenteredShrink` is preferable, to keep a popup true to its approximate intended size.

Reply:
Definitely a good change.